example/migrations/main.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2015 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// migrations command demostrate the functionality of
+// user data migration for authenticated Github user
+// and list all the user migrations.
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+func FetchAllUserMigrations() ([]*github.UserMigration, error) {
+	ctx := context.Background()
+	ts := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: "b453839afb5b7871b9dabfe8184b314747f6523a"},
+	)
+	tc := oauth2.NewClient(ctx, ts)
+	client := github.NewClient(tc)
+
+	user_migrations, _, err := client.Migrations.ListUserMigrations(ctx)
+
```

Comment:
nit: I would nix this blank line to more tightly couple the call with the resulting return value.